policyengine_taxsim/config/variable_mappings.yaml

```diff
@@ -413,25 +413,43 @@ policyengine_to_taxsim:
     group_order: 4
     full_text_group: "State Tax Calculation"
     group_column: 1
-    special_cases:
+    pre_simulation:
       - ms:
-          implemented: false
-          variable: na_pe
+          implemented: true
+          pre_variable: ms_files_separately
+          variables:
```

Comment:
these variables are computed on the person-level so the entity for each of those is "person" while the entity of the other standard deduction variables is "tax_unit"